docs/content/en/contribute/releasing/_index.md

```diff
@@ -6,39 +6,277 @@ weight: 11
 description: "How to perform releases of k8ssandra-operator"
 ---
 
+## Prerequisites
+
+- [yq](https://github.com/mikefarah/yq) installed.
+- [Hugo](https://gohugo.io/documentation/) installed.
 
 ## Major / Minor Release
 
-These steps apply to either creating a major or a minor release.
-They should be performed using the k8ssandra/k8ssandra-operator repo as origin (not a fork).
+### Preliminary steps
+
+Create these environment variables, we'll reference them throughout the rest of this section:
+
+- the Major.minor coordinates of the version we are about to release:
+    ```shell
+    export RELEASED_MAJOR_MINOR=1.6
+    ```
+- the Major.minor coordinates of the next version on the main branch (usually this will be the next minor, but there
+  could also be a major bump, like 2.0):
+    ```shell
+    export NEXT_MAJOR_MINOR=1.7
+    ```
+- the name of your Git remote for k8ssandra/k8ssandra-operator:
+    ```shell
+    export GIT_REMOTE=origin
+    ```
+
+In your local `k8ssandra-operator` clone, check out the main branch and make sure it's up-to-date:
+```shell
+git checkout main
+git pull ${GIT_REMOTE?} main
+```
+
+### Create the release branch
+
+```shell
+git checkout -b release/${RELEASED_MAJOR_MINOR?}
+```
+
+### Update the changelog
+
+Edit `CHANGELOG/CHANGELOG-${RELEASED_MAJOR_MINOR?}.md` and replace the `## unreleased` section with the new release
+version and date (keep the `## unreleased` section above it).
+
+This should look similar to this:
+
+```diff
+diff --git a/CHANGELOG/CHANGELOG-1.6.md b/CHANGELOG/CHANGELOG-1.6.md
+index 92e9647..a42e086 100644
+--- a/CHANGELOG/CHANGELOG-1.6.md
++++ b/CHANGELOG/CHANGELOG-1.6.md
+@@ -15,6 +15,8 @@ When cutting a new release, update the `unreleased` heading to the tag being gen
+
+ ## unreleased
+
++## v1.6.0 - 2023-03-10
++
+ * [CHANGE] [#907](https://github.com/k8ssandra/k8ssandra-operator/issues/907) Update to cass-operator v1.15.0, remove Vector sidecar, instead use cass-operator's server-system-logger Vector agent and only modify its config
+```
+
+Commit the changes:
+```shell
+git add CHANGELOG/CHANGELOG-${RELEASED_MAJOR_MINOR?}.md
+git commit -m"Update changelog for v${RELEASED_MAJOR_MINOR?}.0"
+```
+
+### Check that the documentation builds
+
+```shell
+cd docs
+hugo mod clean
+npm run build:production
+
+cd ..
+```
+
+If there are any errors, fix them and commit them on the release branch.
+
+### Create the release commit
+
+Update the image tag in the default kustomization:
+```shell
+yq e -i '.images[0].newTag="v"+env(RELEASED_MAJOR_MINOR)+".0"' config/deployments/default/kustomization.yaml
```

Comment:
The only downside is that this loses blank lines and comments.
We currently have a couple of blank lines in `kustomization.yaml`, and no comments in either file.